components/brave_wallet_ui/common/hooks/transaction-parser.ts

```diff
@@ -73,7 +73,19 @@ export interface ParsedTransaction extends ParsedTransactionFees {
   minBuyAmount?: string
 }
 
-export function useTransactionFeesParser (selectedNetwork: BraveWallet.NetworkInfo, networkSpotPrice: string, solFeeEstimates?: SolFeeEstimates) {
+export const useTransactionFeesParser = (
+  transactionNetwork?: BraveWallet.NetworkInfo
+) => {
+  // redux
+  const {
+    selectedNetwork: reduxSelectedNetwork,
+    solFeeEstimates
+  } = useSelector(({ wallet }: { wallet: WalletState }) => wallet)
+  const selectedNetwork = transactionNetwork || reduxSelectedNetwork
+
+  // custom hooks
+  const { networkAssetSpotPrice } = usePricing()
```

Comment:
Should this `usePricing` be dependent on the `selectedNetwork` computed above?

Reply:
fixed